README.md

```diff
@@ -67,75 +67,102 @@ Add our dependency your `pom.xml`
 
 Checkout our [examples](./examples/README.md) directory for complete examples of how to use the SDK.
 
-Here is a quickstart you can use in your own project:
+Here is a basic example you can use to get started:
```

Comment:
nit: if we are using the README template engine for this repo then I wouldn't expect to see changes to this file in a PR.  They will be overwritten when the template engine runs.  Not a problem if the changes are identical to the ones in the template but just a thing to be aware of.

Reply:
I did it manually since we had trouble getting the readme updated before and the code has changed enough that I didn't want the example in the readme showing the old code.